docs/modules/ROOT/pages/high-availability-mode.adoc

```diff
@@ -0,0 +1,59 @@
+= Running Highly Availiable Hazelcast clusters
+:description: High Availability Mode guarantees that even if the Kubernetes node or whole availability zone is down (and all related Hazelcast members are terminated), you will not experience any data loss.
+
+Hazelcast offers the Partition Group Configuration and its types are ZONE_AWARE and NODE_AWARE, which forces Hazelcast members to store the given data partition backup inside a member located in a different availability zone or Kubernetes node. To learn about more, see the xref:hazelcast:clusters:partition-group-configuration.adoc[Platform documentation].
+
+When using the ZONE_AWARE and NODE_AWARE partition grouping, a Hazelcast cluster spanning multiple availability zones and nodes should have an equal number of members in each availability zones and node. Otherwise, it results in uneven partition distribution among the members.
```

Comment:
```suggestion
== Using Multiple Availability Zones or Nodes

When using either type of partition grouping (ZONE_AWARE or NODE_AWARE) with a Hazelcast cluster that spans multiple availability zones and nodes, you must have an equal number of members in each zone or node. Otherwise, it results in the uneven distribution of partitions amongst the members.
```

Reply:
The wording seems ok to me but why do we need a separate section for that? Without multiple nodes/zones, this feature is meaningless so if we create a specific section like that it might create a misunderstanding among users that the operator supports one more topology.

Reply:
Agreed.